environment/visibility.csv

```diff
@@ -98,6 +98,7 @@ MCNL,"Excellent, no obstructions",2008-05-27T00:00:00Z,9999-01-01T00:00:00Z
 METH,"Cloud cover made it impossible to see mountains to the west, calculations show < 10 degrees.",2010-09-10T00:00:00Z,9999-01-01T00:00:00Z
 MING,Excellent,2010-04-28T00:00:00Z,9999-01-01T00:00:00Z
 MKNO,Obstructions to East up to 12 degrees formed by Mokai Patea Range.,2011-05-11T00:00:00Z,9999-01-01T00:00:00Z
+MKP1,Distant hills block no more than 5 degrees north. Nearby brush and hills in other directions block up to an estimated 10 - 20 degrees in elevation,2025-03-17T20:32:38Z,9999-01-01T00:00:00Z
```

Comment:
Normally we don't put hours, minutes and seconds in the timestamp in this table.